oauth/origin/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Redirecting to Google</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/svg+xml" href="./favicon.svg" />
+    <style>
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+      body {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-family: Arial, sans-serif;
+        font-size: 14px;
+        line-height: 1.4;
+        text-align: center;
+        padding: 0 1rem;
+      }
+      img.logo {
+        display: block;
+        margin: 0 auto;
+        max-width: 150px;
+      }
+      h1 {
+        margin-top: 0.5rem;
+        font-size: 18px;
+      }
+      p {
+        font-size: 14px;
+      }
+      p.error {
+        color: red;
+        font-weight: bold;
+      }
+    </style>
+  </head>
+  <body>
+    <img class="logo" src="./favicon.svg" alt="Logo" />
+    <h1>Redirecting to Google...</h1>
+
+    <script>
+      (function () {
+        "use strict";
+
+        function displayError(message) {
+          document.body.innerHTML = `<p class="error">${message}</p>`;
```

Comment:
Not sure if the context is available, but it would be really nice for a user to be able to navigate back to the originating application with a button or something similar, but not a blocker